frontend/packages/shared/src/constants.js

```diff
@@ -19,5 +19,6 @@ export const PROD_ENV_TYPE = 'production';
 export const PROTECTED_TASK_NAME_CUSTOM_RECEIPT = 'CustomReceipt';
 export const PREVIEW_MOCK_PARTY_ID = '51001';
 export const PREVIEW_MOCK_INSTANCE_GUID = 'f1e23d45-6789-1bcd-8c34-56789abcdef0';
-
 export const MEDIA_QUERY_MAX_WIDTH = '(max-width: 1024px)';
+export const DATA_MODEL_NAME_MAX_LENGTH = 100;
+export const DATA_MODEL_NAME_REGEX = /^[a-zA-Z][a-zA-Z0-9_\-æÆøØåÅ]*$/;
```

Comment:
I suggest defining these constants closer to where they are used. 
But maybe it is a good idea to have a global constant for `GENERAL_NAME_REGEX` which we can use to define both regEx for data model file names and image file names.

```
**In dataModelleing:**
const DATA_MODEL_NAME_REGEX = GENERAL_NAME_REGEX;

**In uploadImage:**
const IMAGE_NAME_REGEX = GENERAL_NAME_REGEX;
```

Reply:
I can move `DATA_MODEL_NAME_MAX_LENGTH` into `useValidateSchemaName` maybe, but `DATA_MODEL_NAME_REGEX` is now used in both `frontend/app-development/features/dataModelling/SchemaEditorWithToolbar/TopToolbar/utils/validationUtils.ts` and `frontend/packages/shared/src/hooks/useValidateSchemaName.ts`.

And `GENERAL_NAME_REGEX` I think should be its own issue.

Reply:
Hm, I'm actually not sure how to go about on this. Maybe @framitdavid have some input here. I remember him commented something similar on something I was working on some time back 🙈

Reply:
`DATA_MODEL_NAME_MAX_LENGTH` has now been moved into the hook where it is used 😊